src/Modules/UngradedModules/SubquoModuleElem.jl

```diff
@@ -933,7 +933,9 @@ end
 Return the generators of `M`.
 """
 function gens(M::SubquoModule{T}) where T
-  return SubquoModuleElem{T}[gen(M,i) for i=1:ngens(M)]
+  R = base_ring(M)
+  e = R(1)
+  return [SubquoModuleElem{T}(sparse_row(R, [i], [e]; sort=false), M) for i in 1:ngens(M)]
```

Comment:
```suggestion
  return [SubquoModuleElem{T}(sparse_row(R, [i], [e]), M) for i in 1:ngens(M)]
```
no longer needed due to https://github.com/thofma/Hecke.jl/pull/1502